Mlem/Extensions/Swipey Actions.swift

```diff
@@ -62,130 +62,145 @@ struct SwipeyView: ViewModifier {
         self.secondaryLeadingAction = secondaryLeadingAction
         self.primaryTrailingAction = primaryTrailingAction
         self.secondaryTrailingAction = secondaryTrailingAction
-
+        
         // other init
         _leadingSwipeSymbol = State(initialValue: primaryLeadingAction?.symbol.fillName)
         _trailingSwipeSymbol = State(initialValue: primaryTrailingAction?.symbol.fillName)
         _isDragging = isDragging
     }
-
+    
     // swiftlint:disable cyclomatic_complexity
     // swiftlint:disable function_body_length
     func body(content: Content) -> some View {
-            content
-                .offset(x: dragPosition) // using dragPosition so we can apply withAnimation() to it
-                .highPriorityGesture(
-                    DragGesture(minimumDistance: 10, coordinateSpace: .global) // min distance prevents conflict with scrolling drag gesture
-                        .updating($dragState) { value, state, _ in
-                            // this check adds a dead zone to the left side of the screen so it doesn't interfere with navigation
-                            if dragState != .zero || value.location.x > 50 {
-                                state = value.translation.width
-                            }
+        content
+        // add a little shadow under the edge
+            .background {
+                Rectangle()
+                    .foregroundColor(.clear)
+                    .border(width: 10, edges: [.leading, .trailing], color: .black)
+                    .shadow(radius: 5)
+                    .mask(Rectangle().frame(width: UIScreen.main.bounds.width + 20)) // clip top/bottom
```

Comment:
nit: can we start taking things like shadows and making them components so we dont have slightly different variations all over the app? does it make sense for this?